packages/@webex/plugin-meetings/src/meetings/index.ts

```diff
@@ -1374,7 +1375,12 @@ export default class Meetings extends WebexPlugin {
           // (they had a locusUrl previously but are no longer active) in the sync
           for (const meeting of Object.values(meetingsCollection)) {
             // @ts-ignore
-            if (meeting.locusUrl && !activeLocusUrl.includes(meeting.locusUrl)) {
+            const isNotActiveLocusMeeting = !activeLocusUrl.includes(meeting.locusUrl);
+            if (
+              (keepOnlyLocusMeetings && isNotActiveLocusMeeting) ||
+              // @ts-ignore
+              (!keepOnlyLocusMeetings && meeting.locusUrl && isNotActiveLocusMeeting)
+            ) {
```

Comment:
suggestion: This can be simplified:

```ts
if ((keepOnlyLocusMeetings || meeting.locusUrl) && !activeLocusUrl.includes(meeting.locusUrl)) {
```

Which works on the assumption that the active URL array never contains `undefined` as an entry (which it shouldn't!). This is also marginally more efficient, since if we are keeping non-locus meetings, it doesn't even bother checking if it is an active URL or not (the meeting should be kept since it lacks a locusUrl).

Reply:
I like this better! Thank you!